src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ExceptionsGenericError.cs

```diff
@@ -0,0 +1,160 @@
+using System;
+using System.Data;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted
+{
+    public class ExceptionsGenericErrors : IClassFixture<ExceptionGenericErrorFixture> {
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        [ActiveIssue(10036)]
+        public void TestCommandOptionWithNoTceFeature () {
+            SqlConnectionStringBuilder sb = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
+            CertificateUtility.ChangeServerTceSetting (false, sb); // disable TCE on engine.
+            SqlConnection conn = CertificateUtility.GetOpenConnection (false, sb, fSuppressAttestation: true);
```

Comment:
Connection should be added in `using` block. Applicable to all disposable objects in this class in all tests.